lib/writingFile.test.js

```diff
@@ -0,0 +1,21 @@
+const data = require('./data');
```

Comment:
The name of this file should be the same as the file being tested + .test.ts

Like data.test.ts ;)

Reply:
Done :)